src/sentry/integrations/slack/unfurl/issues.py

```diff
@@ -63,10 +65,16 @@ def unfurl_issues(
     return out
 
 
+issue_link_regex = re.compile(
+    r"^https?\://(?#url_prefix)[^/]+/(?#organizations)[^/]+/(?#organization_slug)[^/]+/issues/(?P<issue_id>\d+)(?:/events/(?P<event_id>\w+))?"
```

Comment:
```suggestion
    r"^https?\://(?#url_prefix)[^/]+/organizations/(?#organization_slug)[^/]+/issues/(?P<issue_id>\d+)(?:/events/(?P<event_id>\w+))?"
```

If you want to match `organizations` specifically you could use a literal here.